fwupdmgr: add page (#2666)
tldr-pages/tldr: pages/linux/fwupdmgr.md

# fwupdmgr

> Update the firmware of local devices.

> - See which devices fwupdmgr can update firmware for:

Reviewer: Perhaps this will read better?

```suggestion
- List the devices fwupdmgr can update firmware of:
```

`fwupdmgr get-devices`

> - Refresh update metadata:

Reviewer: What's update metadata?
